FieldOpt/ERTWrapper/CMakeLists.txt

```diff
@@ -12,9 +18,18 @@ add_library(ertwrapper
 add_library(fieldopt::ertwrapper ALIAS ${PROJECT_NAME})
 
 include_directories(${EIGEN3_INCLUDE_DIR})
-target_link_libraries(ertwrapper
-		PUBLIC ${ert_ecl}
-		PUBLIC ${ert_util})
+
+if (_WIN32)
+	target_link_libraries(ertwrapper
+			PUBLIC ${ert_ecl}
+			PUBLIC ${ert_geometry}
+			PUBLIC ${ert_util}
+			PUBLIC ${ZLIB_LIBRARY})
+else()
+	target_link_libraries(ertwrapper
```

Comment:
Not needed in windows either. No documentation whatsoever, that I can find. Here are source files:
CMakeLists.txt
geo_pointset.c
geo_polygon.c
geo_polygon_collection.c
geo_region.c
geo_surface.c
geo_util.c

Nothing we use for wic calculation. Maybe something we can later use for boundary constraint handling, and similar, e.g., point inside polygon function, etc.